albatross.ml

```diff
@@ -60,12 +60,8 @@ module Make (S : Tcpip.Stack.V4V6) = struct
     let* path =
       match domain with
       | None -> Ok Vmm_core.Name.root
-      | Some domain -> (
-          match Vmm_core.Name.Path.of_string domain with
-          | Error (`Msg msg) ->
-              Error
-                (Fmt.str "albatross: domain %s is not a path: %s" domain msg)
-          | Ok path -> Ok (Vmm_core.Name.make_of_path path))
+      | Some domain ->
+          Ok (Vmm_core.Name.make_of_path (Vmm_core.Name.Path.of_label domain))
     in
```

Comment:
This is basically now something that will always succeed, no?

```OCaml
let path = Option.value ~default:Vmm_core.Name.root (Option.map (fun d -> Vmm_core.Name.make_of_path (Vmm_core.Name.Path.of_label d)) domain) in
```

Or is there something I'm missing?